test/e2e/storage/api/BUILD

```diff
@@ -0,0 +1,52 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "driveroperations.go",
+        "snapshotresource.go",
+        "testconfig.go",
+        "testdriver.go",
+        "testpattern.go",
+        "testsuite.go",
+        "volumeresource.go",
+    ],
+    importpath = "k8s.io/kubernetes/test/e2e/storage/api",
```

Comment:
Can we name this package "framework" rather than "api"? api may be confusing with the kubernetes api

Reply:
Done